src/main/javassist/CtClass.java

```diff
@@ -69,7 +69,7 @@
     /**
      * The version number of this release.
      */
-    public static final String version = "3.28.0-GA";
+    public static final String version = "3.27.1-GA";
```

Comment:
do you want to put there h2o specific suffix?

Reply:
good idea, changed to 3.27.1-h2o to make it explicit